src/xdocs/google_style.xml

```diff
@@ -2013,11 +2004,6 @@
                         src="images/ok_blue.png"
                         alt="" />
                   </span>
-                  <a href="checks/blocks/emptyblock.html#EmptyBlock">EmptyBlock</a>
-                  (<a href="https://github.com/search?q=path%3Asrc%2Fmain%2Fresources%20path%3A**%2Fgoogle_checks.xml+repo%3Acheckstyle%2Fcheckstyle+EmptyBlock">
-                  config</a>)
-                  <br />
-                  <br />
```

Comment:
we didn't required EmptyBlock for this section ( [6.2](https://google.github.io/styleguide/javaguide.html#s6.2-caught-exceptions) ), EmptyBlock and section 6.2 were not related to each ohter so I removed it